frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -87,9 +92,27 @@ export const useDashboardUrlParams = ({
     titled,
     font,
     setFont,
-    theme,
+    theme: normalizedTheme,
     setTheme,
     hideParameters: hide_parameters,
     hideDownloadButton: hide_download_button,
   };
 };
+
+/**
+ * When both `background: false` and `theme: "transparent"` options are supplied,
+ * the new behavior takes precedence (metabase#43838)
+ */
+function normalizeTheme({
```

Comment:
(at first glance I thought this was normalizing the Mantine theme object, but I couldn't think of a better name at the moment.)